src/ripple/core/JobQueue.h

```diff
@@ -184,6 +184,11 @@ class JobQueue
     void
     rendezvous();
 
+    int xxx() const
```

Comment:
Fixed (squashed into commit) and also updated multiplier to be `4` instead of `3` to allow slight oversubscription of cores for better utilization.